packages/core/logging/core-logging-server-internal/BUILD.bazel

```diff
@@ -50,10 +52,12 @@ TYPES_DEPS = [
   "@npm//rxjs",
   "@npm//@types/moment-timezone",
   "@npm//elastic-apm-node",
+  "@npm//chalk",
```

Comment:
How did we migrate the logging service to packages without specifying `@npm//chalk` in the `BUILD.bazel` config? Even on `main` right now, there aren't any errors. 😕

Reply:
Yea, I asked myself the same question... No idea to be honest.